app-backend/common/src/main/scala/S3.scala

```diff
@@ -104,6 +104,35 @@ package object S3 {
     get(client.listObjects(listObjectsRequest), Nil)
   }
 
+  final def getObjectPaths(source: URI): List[String] = {
```

Comment:
: D Lots of S3 objects, but it's a separate issue: https://github.com/azavea/raster-foundry/issues/1822